server/moodle/mod/livequiz/quizcreator/Status.md

```diff
@@ -0,0 +1,38 @@
+
+
+# Status på front end
+
+## Quiz creator page
+
+### Modal
+-[x] Modal kan tage imod billeder
```

Comment:
sørg for ikke at blande dansk og engelsk

Reply:
Det er en midlertidig fil ment til devsne, så de kan se status på den side.
Den skal slettets så snart vi er færdige.

Reply:
hvis vi bliver ved med at tilføje ting der skal fjernes når vi er færdige burde vi så ikke have en liste eller metode for at kunne holde styr eller finde dem igen bag efter?